test/jdk/java/nio/charset/StandardCharsets/Standard.java

```diff
@@ -41,7 +41,8 @@ public class Standard {
 
     private final static String standardCharsets[] = {
         "US-ASCII", "ISO-8859-1", "UTF-8",
-        "UTF-16BE", "UTF-16LE", "UTF-16" };
+        "UTF-16BE", "UTF-16LE", "UTF-16",
+        "UTF-32BE", "UTF-32LE", "UTF-32" };
```

Comment:
Maybe add a newline after the last item?
```suggestion
        "UTF-32BE", "UTF-32LE", "UTF-32",
    };
```